Added xRunbookDirectory

dcrreynolds opened the pull request:
Import Runbook(s) to an SMA instance

dcrreynolds committed:
Adding xRunbookDirectory

dcrreynolds committed:
Updating README.md resources list

dcrreynolds committed:
Fixed some grammer

msftclas commented:
Hi __@dcrreynolds__, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <p>
        It looks like you're a Microsoft contributor (David Reynolds). If you're full-time, we DON'T require a Contribution License Agreement. If you are a vendor, please DO sign the electronic Contribution License Agreement. It will take 2 minutes and there's no faxing! https://cla.microsoft.com.
    </p>

TTYL, MSBOT;

dcrreynolds committed:
Fixed failed test, added Example

Fixed failed test, added Example, and added localization data

msftclas commented:
Hi __@dcrreynolds__, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <p>
        It looks like you're a Microsoft contributor (David Reynolds). If you're full-time, we DON'T require a Contribution License Agreement. If you are a vendor, please DO sign the electronic Contribution License Agreement. It will take 2 minutes and there's no faxing! https://cla.microsoft.com.
    </p>

TTYL, MSBOT;

dcrreynolds committed:
Fixed UTF8 issue

dcrreynolds committed:
Added missed localization String, Get-Item Filter

KarolKaczmarek commented on README.md:
```diff
@@ -71,9 +73,20 @@ Please check out common DSC Resources [contributing guidelines](https://github.c
 
+### Unreleased
+
+* Added new resource to manage a single or directory of Runbooks.
```
please mention it's name

dcrreynolds committed:
Updated unit test, mof, and documentation per peer review.

Unit test now use unit test template. Fixed style issue adding newline
at end of test, mof now includes descriptions, WebServiceEnpoint is now
a key parameter,  and documentation updated to include resource name.

dcrreynolds commented:
Thank you for the feedback! I have updated everything and included feedback received from other peer reviews.

dcrreynolds commented on DSCResources/MSFT_xRunbookDirectory/MSFT_xRunbookDirectory.schema.mof:
```diff
@@ -0,0 +1,10 @@
+class MSFT_xRunbookDirectory : OMI_BaseResource
+{
+    [Key, Description("Path to Runbook(s) to be imported. Accepts wildcards.")] String RunbookPath;
+    [Write, Description("Describes the published state of the imported Runbook(s). Defaults to true, publishing the Runbook.")] Boolean Publish;
```
For runbooks, this could be 3 states if we wanted to apply the Ensure pattern; Absent, Draft, Published. Does having 3 states violate the pattern?

dcrreynolds committed:
Added Ensure support

dcrreynolds committed:
Fixed merge conflict

dcrreynolds committed:
Updating ReadMe

dcrreynolds committed:
Fixed xSCSMAWebServiceServerSetup section header to be consistnt

dcrreynolds committed:
Merge branch 'PowerShell/dev' into dev
